doc/source/feature_comparison.rst

```diff
@@ -20,111 +20,260 @@
+      - Yes
+      - Yes
+    * - MyRocks Storage Engine
+      - Yes
+      - No
+    * - TokuDB Storage Engine
+      - Yes
+      - No
+
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{5cm}|
+
+.. list-table::
+   :header-rows: 1
+   
+   * - Extra Features for Developers
+     - Percona Server 5.7.24
+     - MySQL 5.7.24
+   * - NoSQL Socket-Level Interface
+     - Yes
+     - Yes
+   * - X API Support
+     - Yes
+     - Yes
+   * - InnoDB Full-Text Search Improvements
+     - Yes
+     - No
+   * - Extra Hash/Digest Functions
+     - Yes
+     - No
+
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{5cm}|
+
+.. list-table::
+   :header-rows: 1
+
+   * - Instrumentation and Troubleshooting Features
+     - Percona Server 5.7.24
+     - MySQL 5.7.24
+   * - INFORMATION_SCHEMA Tables
+     - 93
+     - 61
+   * - Global Performance and Status Counters
+     - 789
+     - 357
+   * - Per-Table Performance Counters
+     - Yes
+     - No
+   * - Per-Index Performance Counters
+     - Yes
+     - No
+   * - Per-User Performance Counters
+     - Yes
+     - No
+   * - Per-Client Performance Counters
+     - Yes
+     - No
+   * - Per-Thread Performance Counters
+     - Yes
+     - No
+   * - Global Query Reponse Time Statistics
+     - Yes
+     - No
+   * - Enhanced SHOW ENGINE INNODB STATUS
+     - Yes
+     - No
+   * - Undo Segment Information
+     - Yes
+     - No
+   * - Temporary Tables Information
+     - Yes
+     - No
+   * - Extended Slow Query Logging
+     - Yes
+     - No
+   * - User Statistics
+     - Yes
+     - No
+
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{5cm}|
+
+.. list-table::
+   :header-rows: 1
+
+   * - Performance and Scalability Features
+     - Percona Server 5.7.24
+     - MySQL 5.7.24
+   * - Improved scalability by splitting mutexes
+     - Yes
+     - No
+   * - Improved MEMORY Storage Engine
+     - Yes
+     - No
+   * - Improved Flushing
+     - Yes
+     - No
+   * - Parallel Doublewrite Buffer
+     - Yes
+     - No
+   * - Configurable Page Sizes
+     - Yes
+     - Yes
+   * - Configurable Fast Index Creation
+     - Yes
+     - No
+   * - Per-column Compression for VARCHAR/BLOB and JSON
+     - Yes
+     - No
+   * - Compressed columns with Dictionaries
+     - Yes
+     - No
+
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{5cm}|
+
+.. list-table::
+   :header-rows: 1
+
+   * - Security Features
+     - Percona Server 5.7.24
+     - MySQL 5.7.24
+   * - PAM Authentication Plugin
+     - Yes
+     - Enterprise-Only
+   * - Audit Logging Plugin
+     - Yes
+     - Enterprise-only
+
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{5cm}|
+
+.. list-table::
+   :header-rows: 1
+
+   * - Encryption Features
+     - Percona Server 5.7.24
+     - MySQL 5.7.24
+   * - Encrypt InnodDB data
+     - Yes
+     - Yes
+   * - Encrypt InnoDB tablespaces
+     - Yes
+     - No
+   * - Encrypt InnoDB logs
+     - Yes
+     - No
+   * - Encrypt Binary logs
+     - Yes
+     - No
+   * - Encrypt temporary files
+     - Yes
+     - No
+   * - Key Rotation with scrubbing
```

Comment:
Scrubbing - I wouldn't even call it experimental yet. Also, key rotation and scrubbing are separate things. For several other features here (log encryption for example): these are experimental, not yet GA, I don't think we should write a definite "yes" here.

Reply:
Updated with Experimental.

Reply:
Key Rotation has its own row. Scrubbing has its own row.